src/app.render.js

```diff
@@ -0,0 +1,58 @@
+'use strict'
+var langTerms = require('./lang-terms.js'),
+	responsiveElement = require('./responsive-element.js'),
+	BREAKPOINTS = [
+		{ width: 957, type: "max-width", class: "bp-1" },

```

Comment:
It looks like we are mostly using single quoted strings. Can we be consistent, at least across files?